src/qililab/qprogram/qblox_compiler.py

```diff
@@ -130,12 +133,13 @@ def __init__(self) -> None:
         self._buses: dict[str, BusCompilationInfo]
         self._sync_counter: int
 
-    def compile(
+    def compile(  # noqa: C901
```

Comment:
Pylint skip lines are ok if you do not manage to get around it, but please include the text instead of the code so a developer looking at the code knows immediately what it is about ;)

Reply:
Flake8 does not allow to ignore the error with the sae texts as pylint but I changed the complexity with `noqa: max-complexity=25`

Reply:
I see, thought it was pylint.